src/points/slice.ts

```diff
@@ -11,16 +12,27 @@ interface GetPointsHistoryStartedAction {
   getNextPage: boolean
 }
 
+export type PointsConfig = {
+  activitiesById: {
+    [activityId in PointsActivity]?: {
+      points: number
```

Comment:
Did we decide on [pointsAmount](https://valora-app.slack.com/archives/C029Z1QMD7B/p1712928606786159) for this?

Reply:
yes! i was planning to make the renaming changes in one go but it makes sense to do this one to avoid the need for a redux migration

Reply:
(i'll need to make this change also in the getPointsConfig CF, in case you're wondering why this branch doesn't work anymore)

Reply:
gotcha, sgtm!